customer-information/src/DummyData.js

```diff
@@ -0,0 +1,46 @@
+export const dummyData = {
```

Comment:
Still needed?

Reply:
Left it there so that's its easier to follow the guide.